### PR TITLE
Refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by mikemtzp**
# For this feature we:

- Used a class to create books.
- Added an `add` and `remove` method inside of the class Books.
- Modified the HTML structure to fit the second part of the wireframe
- Add styling to the CSS

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -16,51 +16,55 @@ function generateId() {
 }
 
 function storeData() {
-  localStorage.setItem('books', JSON.stringify(bookList));
+  localStorage.setItem('bookList', JSON.stringify(bookList));
 }
```
Comment (youmari) [inline main.js:20]: - [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that

File: main.js [hunk 2]
```diff
@@ -16,51 +16,55 @@ function generateId() {
...
   bookListElement.id = 'bookList';
   bookList.forEach((book) => {
-    const bookElem = createBook(book);
-    bookListElement.appendChild(bookElem);
+    const bookElem = new Books(book.title, book.author, book.id);
+    const bookElement = bookElem.createBook(bookElem);
+    bookSection.append(bookElement);
   });
-  bookSection.appendChild(bookListElement);
 }
```
Comment (youmari) [inline main.js:68]: - [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that

File: main.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 function initiateBooksData() {
```
Comment (youmari) [inline main.js:1]: - [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @mikemtzp @fibini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linters error.
✔️ Details PR title and description.
✔️ Good commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mikemtzp** [comment]
# Thank you very much for your review
It really helped us to improve our project 😁 
Now:
- [ ] Both members of the team have contributed to the project.
- [ ] The `functions` have been refactored as methods inside of a `class`.

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey Team!  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**